en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/V/Vicious_Mockery.md

```diff
@@ -9,6 +9,6 @@ _Enchantment cantrip_
 **Duration:** Instantaneous
 
 You verbally insult or mock one creature so viciously its mind is seared.
-As long as the target hears you (understanding your words is not required) it takes `1d6` psychic damage and has disadvantage on the first attack roll it makes before the end of its next turn.
+As long as the target hears you (understanding your words is not required) it takes `1d6` psychic damage and has disadvantage on the first attack roll it makes before the end of its next turn if it fails a Wisdom saving throw.
```

Comment:
The wording from WotC here is:
```markdown
If the target can hear you (though it need not understand you), it must succeed on a Wisdom saving throw or take `1d4` psychic damage and have disadvantage on the next attack roll it makes before the end of its next turn.
```
It's clearer in my opinion since a successful save more clearly negates both effects. Plus: it's 4 characters shorter than what you've written. Should we go with their version of this sentence?

(The first sentence with flavour text is still A5e, and that's shorter than WotC, so we can leave that as is.)